# /gazebo_ros_control/pid_gains:

    joint_state_controller:
        type: joint_state_controller/JointStateController
        publish_rate: 50

    wheel_1_joint_velocity_controller:
        type: velocity_controllers/JointVelocityController
        joint: wheel_1_joint
        pid: {p: 100, i: 0.01, d: 10.0}

    wheel_2_joint_velocity_controller:
        type: velocity_controllers/JointVelocityController
        joint: wheel_2_joint
        pid: {p: 100, i: 0.01, d: 10.0}

    wheel_3_joint_velocity_controller:
        type: velocity_controllers/JointVelocityController
        joint: wheel_3_joint
        pid: {p: 100, i: 0.01, d: 10.0}

    wheel_4_joint_velocity_controller:
        type: velocity_controllers/JointVelocityController
        joint: wheel_4_joint
        pid: {p: 100, i: 0.01, d: 10.0}

